lab-ken/README.md

```diff
@@ -0,0 +1 @@
+<p> This is the first lab. I will be writing my first test for Day 1 of code 401 JS.</p>
```

Comment:
Remember to write things in your README according to the Documentation instructions of your labs.